### PR TITLE
feat(flake8): use flake8 to lint files

### PR DESCRIPTION
**Pull request by LittleWhiteYA**
## Summary

- 看 circleci 好像有要引入 flake8，先試改一些檔案

### CODE REVIEW

**Comment by noobOriented** [comment]
Nice

File: backend/.flake8 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 88
```
Comment (noobOriented) [inline backend/.flake8:2]: 88 是怎決定的？

File: backend/.flake8 [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+ignore =
+    E501, # line too long
```
Comment (noobOriented) [inline backend/.flake8:4]: ... 所以設 88?

File: backend/.flake8 (hunk 1 above)
Comment (LittleWhiteYA) [reply backend/.flake8:2]: black default line length

**Comment by stegben** [comment]
Nice! It would be great if you could help adding a `backend-lint` step for CI

**Pull request merged**